assets/terraform-modules/packet/flatcar-linux/kubernetes/controllers.tf

```diff
@@ -75,13 +75,11 @@ data "ct_config" "controller-ignitions" {
   snippets = var.controller_clc_snippets
 }
 
-data "template_file" "etcds" {
+data "ct_config" "etcds" {
```

Comment:
What, why do we use `ct_config` for it? Let's break down what we need to generate and figure it out only with Terraform syntax, that should be possible.

Reply:
Yes, I figured it out after seeing the error. Fixing it